src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -252,6 +258,24 @@ const VisitorSpaceSearchPage: FC = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [isKeyUser, isPublicCollection]);
 
+	useEffect(() => {
+		// Ward: wait until items are rendered on the screen before scrolling
+		if (
+			lastScrollPosition &&
+			lastScrollPosition.page === ROUTES.search &&
+			searchResults?.items
+		) {
+			setTimeout(() => {
+				const item = document.getElementById(
+					`${lastScrollPosition.itemId}`
+				) as HTMLElement | null;
+
+				item?.scrollIntoView({ block: 'center', behavior: 'smooth' });
+			}, 100);
+			dispatch(setLastScrollPosition({ itemId: '', page: ROUTES.search }));
+		}
+	}, [searchResults?.items]);
```

Comment:
shouldn't this array also contain lastScrollPosition?

Reply:
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/113892598/233957916-19e83a3c-5edf-4e1c-8352-faa2a4e8a1f0.png">
I get this error than